scripts/services/c-south

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Run a FogLAMP south service written in C/C++
+if [ "${FOGLAMP_ROOT}" = "" ]; then
+	FOGLAMP_ROOT=/usr/local/foglamp
+fi
+
+if [ ! -d "${FOGLAMP_ROOT}" ]; then
+	logger "FogLAMP home directory missing or incorrectly set environment"
+	exit 1
+fi
+
+# We run the Python code from the python directory
```

Comment:
Please remove references to running python, it will only confuse people in the future.